Sources/Automerge/Document.swift

```diff
@@ -596,12 +620,36 @@ public final class Document: @unchecked Sendable {
 
     /// Establish a cursor at the position and point of time you specify in the list or text object you provide.
     ///
+    /// In collaborative applications, maintaining stable cursor positions is crucial. Traditional index-based
```

Comment:
^^ I'd like to see the same grammar and language tweaks in this documentation set as well. The changes are mostly aimed at removing passive voice and being explicit about what's doing the changing or tracking.

Reply:
Indeed, your recent grammar tweaks have rendered it redundant, so we can remove it for clarity. Addressed in: https://github.com/automerge/automerge-swift/commit/f7657f61896f70ca1d6dc36b74355274daf20b8a